src/elastio-snap.c

```diff
@@ -38,7 +38,7 @@ MODULE_VERSION(ELASTIO_SNAP_VERSION);
 #include <uapi/linux/mount.h>
 #endif
 
-#if defined HAVE_BLK_ALLOC_QUEUE_MK_REQ_FN_NODE_ID || defined USE_BDOPS_SUBMIT_BIO
+#if defined HAVE_BLK_ALLOC_QUEUE_MK_REQ_FN_NODE_ID || defined USE_BDOPS_SUBMIT_BIO || defined HAVE_BLK_MQ_MAKE_REQUEST
```

Comment:
```suggestion
#if defined HAVE_BLK_MQ_MAKE_REQUEST || defined USE_BDOPS_SUBMIT_BIO
```
Better to replace all usages of the `HAVE_BLK_ALLOC_QUEUE_MK_REQ_FN_NODE_ID` to the `HAVE_BLK_MQ_MAKE_REQUEST` except the single place, where  `HAVE_BLK_ALLOC_QUEUE_MK_REQ_FN_NODE_ID` is used around appropriate function call https://github.com/elastio/elastio-snap/blob/162d2640ae468ccaee43481a837504930ead5c0e/src/elastio-snap.c#L3829-L3832
That's great that you introduced `HAVE_BLK_MQ_MAKE_REQUEST` feature test and appropriate define. Let's use it where it's applicable.